### DISCUSSION TITLE
API - Very limited stakes

### DISCUSSION THREAD

**Discussion by Markt42** [cloudbet/docs#55] [Q&A]
Hi Cloudbet team,

I have been betting using the API for the past couple of weeks. Since two days ago, it seems that the stakes on all of my matches have been limited very heavily but I am still allowed to bet on all matches. For all my bets, I satisfy the constraint that my bet value should be between minStake and maxStake (assuming these are in EUR/USD) from /v2/odds/lines, but I cannot come close to the value maxStake in any of my bets. 
Reading the other discussions on this forum, I guess that it can be the case that I did not adhere to the following constraint from the general terms and conditions: 
_• When using the API be considerate. If the bet rejection rate on the betting API is too high (e.g. > 80% on your last 100 bets) your account will be flagged for abusive behavior. You may get blocked for up to 7 days for integrity checks._

I do receive the status of "PRICE_ABOVE_MARKET" occasionally, and I think this might be in more than 20% of my bets, however this is not something I was monitoring. But as this condition states that I might get **blocked**, I thought it was odd that I am still allowed to place bets, just of much lower values. As an example, I am trying to bet 89.96 ADA on the away side of 16359318:'c140c6f-adrian-eliasz-v-c15135d-tadeusz-piotrowski' [event_id:event_key] in table tennis. According to the /v2/odds/lines endpoint, I am allowed to bet 360.15 (EUR?) on this match. However, in the response to this bet, I receive a max stake of 2.83776 ADA with status: STAKE_ABOVE_MAX. This value is much lower than the maxStake I receive from the /v2/odds/lines endpoint.
As I assume I might be blocked, I stopped my application for now. My question is now: Is this behavior in line with what you would expect with a blocked account? If this is the case, I will more closely monitor the rejection rate of my bets and hope this will not happen again in the future. If you suspect it is something else, I would love to hear from you as well. 

Thanks in advance for the help!

**Comment by kgravenreuth** [comment]
Hi @Markt42 

Thanks for the detailed report about the issue you observed. 

In such cases, the limits are controlled by our risk management system and could be dynamically changed for your player. The limits could potentially be increased over time with more betting activity, but this is not a guarantee and depends on our risk management system's assessment. 

Please let us know if you have any technical questions around the Cloudbet API and we would be happy to help. 

Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Reply by Markt42** [reply]
Hi Klaus,

Thanks for your quick reply to my question. So if I understand correctly, the issue is probably not because my account is blocked but due to an internal Cloudbet system limiting my stakes. Will my account then be reviewed to see if my stakes can be increased again? Or can I contact someone to ask if my stakes can be increased? Because at the moment, the allowed stake on my bets is so low it is not really usable for me. 

Thanks again for the help!

**Reply by jhulka** [reply]
What the progress? Are you still limited?

**Reply by kgravenreuth** [reply]
Hi @jhulka ,
As I mentioned above, limits are controlled by our risk management system, and could potentially be increased over time with more betting activity, but this is not a guarantee.

If you have any technical questions, please let us know.

Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
